### PR TITLE
Imagec failing with new flags containing dashes

### LINKED ISSUE #956: imagec failing with new insecure flags
The insecure flag in imagec was split into two new flags:

insecure-skip-verify
insecure-allow-http

The dashes in the flag name cause flag.parse to stop at the first dash instance and seeing a flag of "insecure" that no longer exists.

### PR DESCRIPTION
**Pull request by cgtexmex**
Updated the new insecure flags to use an underscore.

Fixes #956

### CODE REVIEW

File: cmd/imagec/imagec.go [hunk 1]
```diff
@@ -146,8 +146,8 @@ func init() {
 
 	flag.BoolVar(&options.stdout, "stdout", false, i18n.T("Enable writing to stdout"))
 	flag.BoolVar(&options.debug, "debug", false, i18n.T("Show debug logging"))
-	flag.BoolVar(&options.insecureSkipVerify, "insecure-skip-verify", false, i18n.T("Don't verify certificates when fetching images"))
-	flag.BoolVar(&options.insecureAllowHTTP, "insecure-allow-http", false, i18n.T("Uses unencrypted connections when fetching images"))
+	flag.BoolVar(&options.insecureSkipVerify, "insecure_skip_verify", false, i18n.T("Don't verify certificates when fetching images"))
+	flag.BoolVar(&options.insecureAllowHTTP, "insecure_allow_http", false, i18n.T("Uses unencrypted connections when fetching images"))
```
Comment (fdawg4l) [inline cmd/imagec/imagec.go]: Dash with underscores are not my favorite.  Can you nuke the underscores altogether?

Comment (cgtexmex) [inline cmd/imagec/imagec.go]: yep..I've got no vested interested in underscores...just makes a long flag name.

**Comment by hmahmood** [comment]
Is this still a problem if we invoke imagec using -- instead - for the options?

File: cmd/imagec/imagec.go (hunk 1 above)
Comment (fdawg4l) [inline cmd/imagec/imagec.go]: All are fine with me.  My only issue is with underscore.  Maybe @caglar10ur can weigh in.

Comment (fdawg4l) [inline cmd/imagec/imagec.go]: I'm hearing things about this not being the issue.  I'll let @hickeng and @caglar10ur explain.

Comment (cgtexmex) [inline cmd/imagec/imagec.go]: I shall wait to be enlightened...

**Comment by cgtexmex** [comment]
@hmahmood I still see the issue regardless of single or double dash for flag identifier...

**Comment by caglar10ur** [comment]
Stop the press the issue is something else (PR coming)

**Comment by caglar10ur** [comment]
closing this in favor of https://github.com/vmware/vic/pull/959

**Pull request closed without merging**